hawk/app/views/tickets/show.html.haml

```diff
@@ -4,59 +4,110 @@
       = icon_tag "search"
 
       = _("Ticket:")
-      = @ticket.id
-
-      %small
-        = @ticket.ticket
+      = @ticket[:id]
 
   .modal-body
-    %h4
-      = _("Properties")
-
-    %table.table.table-striped
-      %tbody
-        %tr
-          %th.col-xs-4
-            = _("Granted")
-          %td.col-xs-8
-            - if @ticket.granted
-              = _("Yes")
-            - else
-              = _("No")
-        %tr
-          %th.col-xs-4
-            = _("Loss Policy")
-          %td.col-xs-8
-            - if @ticket.loss_policy.blank?
-              = _("None")
-            - else
-              = @ticket.loss_policy
-    %h4
-      = _("Resources")
+    .panel.panel-default
+      .panel-heading#t-properties{role: :tab}
+        %h4.panel-title
+          %a{role: :button, data: {toggle: :collapse}, href: "#t-properties-body", aria: {expanded: false, controls: "t-properties-body"}}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [138/80]